datafusion/core/tests/fuzz_cases/aggregation_fuzzer/fuzzer.rs

```diff
@@ -132,7 +133,17 @@ struct QueryGroup {
 }
 
 impl AggregationFuzzer {
+    /// Run the fuzzer, printing an error and panicking if any of the tasks fail
     pub async fn run(&self) {
+        let res = self.run_inner().await;
+
+        if let Err(e) = res {
+            println!("{e}");
```

Comment:
do we need this block?

Reply:
Without that block the error is printed with embedded newlines (so like "Error\nfoo\nbar\n..") which is hard to understand. 

I will make the rationale clear with a comment

```suggestion
            // Print the error via `Display` so that it displays nicely (the default `unwrap()` 
            // prints using `Debug` which escapes newlines, and makes multi-line messages
            // hard to read
            println!("{e}");
```